templates/user.twig

```diff
@@ -12,15 +20,25 @@
 					</div>
 					<div class="caption">
 						<h3>{{ user.username }}</h3>
-						<p>Joined {{ user.joined | date('M j, Y') }}</p>
-						<p>database does not have such fields lol</p>
+						<p>Joined {{ user.joined | date('F jS, Y') }}</p>
+						<p>{{ user.description }}</p>
 						<p>
 							<a href="#" class="btn btn-warning" role="button">Subscribe</a>  
-							<a href="#"class="btn btn-secondary" role="button">Add friend</a>
+							<a href="#"class="btn btn-secondary" role="button">Friend</a>
 						</p>
 					</div>
 				</div>
 			</div>
 		</div>
+		<div class="col-md-6 col-lg-8">
```

Comment:
shouldn't be horizontal, semantic UI-era poktube had this exact same issue

Reply:
nonono!